core/src/main/java/org/infinispan/commands/write/WriteCommand.java

```diff
@@ -78,4 +78,26 @@ default boolean isWriteOnly() {
     * @return the {@link CommandInvocationId} associated to the command.
     */
    CommandInvocationId getCommandInvocationId();
+
+   /**
+    * Initializes the {@link BackupWriteRpcCommand} to send the update to backup owner of a key.
+    * <p>
+    * This method will be invoked in the primary owner only.
+    *
+    * @param command the {@link BackupWriteRpcCommand} to initialize.
+    */
+   default void initBackupWriteRpcCommand(BackupWriteRpcCommand command) {
```

Comment:
Because I tried to merge BackupWriteRpcCommand and BackupMultiKeyWriteRpcCommand and WriteCommand was the common interface to both. Then I forget to revert it :+1:

Reply:
actually, I forgot about PutMapCommand. The PutMapCommand doesn't extends AbstractWriteManyCommand

Reply:
Ah... And are you sure you want to initialize the backup commands directly from the command, rather than using a set of helpers? You could just add another method to `WriteManyCommandHelper`.

Reply:
I disagree with the helper. IMO, the backup command is tightly coupled to the write command. After this PR is integrated, (as they are missing command to use triangle) I would rather extend the write command interface as follow:
```
BackupWriteCommand toBackupWriteCommand(Collection<?> keys);
```

Reply:
OK; My take on this is that the Backup* commands are an implementation detail of triangle interceptors, but be it your way.

Not sure why you want to postpone this after this PR? I find two distinct methods in such base interface inelegant.

Reply:
I can create a new Interface, (`TriangleWriteCommand`?) to put the signature if you think it would be cleaner. 
I'll do it after this PR because it isn't related. I don't want to double the size of the PR if I don't need to :)

Reply:
Eh, no, not another interface... That wouldn't unbind the commands. Let's keep this, then...